src/controller/login.js

```diff
@@ -1,51 +1,119 @@
-import Layout from "component/layout";
+import ModelAccount from "model/account";
+import ModelUser from "model/user";
+import AppRequest from "component/request";
 
-var LoginController = {
+import LoginView from "view/login";
+
+import LocalForage from "localforage";
+import Storage from "model/storage";
+
+export default {
     controller: function() {
-        var storage = window.localStorage;
-        return {
-            storage: storage,
-            data: {
-                token: m.prop(LoginController.getToken(storage)),
-                endpoint: m.prop(LoginController.getEndpoint(storage))
+
+        this.accounts = m.prop([]);
+        this.users = m.prop([]);
+        this.endpoint = m.prop("https://api.wheniwork.com/2");
```

Comment:
It will be a constant in the future, but for testing, it's nice to change frequently without having to rebuild.